endorsement/storeapi.proto

```diff
@@ -53,5 +123,13 @@ service Fetcher {
     rpc GetEndorsements(GetEndorsementsArgs) returns (GetEndorsementsResponse);
 }
 
-
+// Provisioner is the Service end point that is used to provision
+// Software Components and Trust Anchors to and from
+// the Endorsement Store
```

Comment:
"provision from" is not correct.  In general, I don't think these comments are very helpful.  I suggest we just drop them.